lib/src/main/java/org/exoplatform/social/addons/search/ProfileSearchConnector.java

```diff
@@ -254,10 +254,10 @@ private String buildTypeEx(Type type) {
         result = "connections";
         break;
       case INCOMING:
-        result = "incomings";
+        result = "outgoings";
         break;
       case OUTGOING:
-        result = "outgoings";
+        result = "incomings";
```

Comment:
Does not look very logical :) Not a mistake ? Can you explain ?

Also, we should probably have an enum here with a value : CONFIRMED("connections")